app/reducers/snackbar.js

```diff
@@ -289,11 +291,15 @@ const messages = defineMessages({
   TRZ_GETWALLETCREATIONMASTERPUBKEY_FAILED: {
     id: "trezor.getWalletCreationMasterPubKey.failed",
     defaultMessage: "Failed to obtain master extended pubkey from trezor device: {originalError}"
+  },
+  ERROR_IS_OBJECT: {
+    id: "general.error",
```

Comment:
It's probably better to use "snackbar.errorObject" or something  more specific rather than "general.error".